tests/lib/Setup.pm

```diff
@@ -98,7 +101,7 @@ sub nrQueuedBuildsForJobset {
 }
 
 sub createBaseJobset {
-    my ($jobsetName, $nixexprpath) = @_;
+    my ($jobsetName, $nixexprpath, $jobspath) = @_;
```

Comment:
I wasn't keen on this change: it seemed better to set the jobs path in a local variable in this module and refer to it from here. However, I could imagine a future where we're making a per-test jobs directory, and so this would be interesting.